docs/dev/manual_testing.rst

```diff
@@ -0,0 +1,60 @@
+
+Manual Testing & QA
+===================
+
+
+Cross-browser and OS Testing
+----------------------------
+
+It's vital to ensure that our app works across a wide range of browsers and operating systems, particularly older versions of Windows and Android that are common on older and cheaper devices.
+
+In particular, we want to ensure that Kolibri runs on major browsers that match any of `the following criteria <http://browserl.ist/?q=%3E+1%25%2C+last+2+versions%2C+ie+%3E%3D+9%2C+Firefox+ESR>`_:
+
+ * within the last two versions
+ * IE 9+ on Windows XP and up
+ * has at least 1% of global usage stats
+
+Here are some useful options, in order of simplicity:
+
+**BrowserStack**
+
+`BrowserStack <https://www.browserstack.com/>`_ is an incredibly useful tool for cross-browser and OS testing. In particular, it's easy to install plugin which forwards ``localhost`` to a VM running on their servers, which in turn is displayed in your browser.
+
+**Amazon Workspaces**
+
+In some situations, simply having a browser is not enough. For example, a developer may need to test Windows-specific backend or installer code from another OS. In many situations, a virtual machine is appropriate - however these can be slow to download and run.
+
+Amazon's `AWS Workspaces <https://aws.amazon.com/workspaces/>`_ provides a faster alternative. They run Windows VMs in their cloud, and developers can RDP in.
+
+**Local Virtual Machines**
+
+Workspaces is very useful, but it has limitations: only a small range of OSes are available, and connectivity and provisioning are required. An alternative is to run the guest operating system inside a virtual machine using e.g. `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_. This also gives more developer flexibility, including e.g. shared directories between the guest and host systems.
+
```

Comment:
I just remembered I wrote this[ tutorial for testing KA Lite in VirtualBox](https://docs.google.com/document/d/10LgeCJmqsweui0yTTCDf4DjY5aoNNpXG8hF_DGKUHAI/edit), back when I started the QA. It is in our FLE Shared folder, but it is viewable to general public...